tests/test_create_climo_files.py

```diff
@@ -158,22 +162,46 @@ def test_climo_metadata(outdir, tiny_dataset, operation, t_start, t_end, split_v
 
     suffix = {
         'std': 'SD',
-        'mean': 'Mean'
+        'mean': 'Mean',
+        'sum': 'Mean'
     }[operation]
 
     if split_intervals:
         assert frequencies == {
            'daily': {'mClim' + suffix, 'sClim' + suffix, 'aClim' + suffix},
            'monthly': {'sClim' + suffix, 'aClim' + suffix},
```

Comment:
This isn't something that you changed but shouldn't this be `'monthly': {'mClim' + suffix, 'aClim' + suffix}`?

Reply:
Hmm. When we have an input dataset with a time resolution of <= monthly, then we form monthly, seasonal, and annual climo means, yes? Which means that the monthly case should be `'monthly': {'mClim' + suffix, 'sClim' + suffix, 'aClim' + suffix}`.

Reply:
The code that generates it is as follows, and I believe is correct. So this is a testing issue, not a script issue.

```
prefix = ''.join(abbr for interval, abbr in (('monthly', 'm'), ('seasonal', 's'), ('annual', 'a'), )
                         if interval in interval_set)
        cf.frequency = prefix + 'Clim' + suffix
```

where `interval_set` is a subset of {'monthly', 'seasonal', 'annual'} as determined by the file's actual contents.